### DISCUSSION TITLE
[Lesson9] failed parsing.. issue while `make deploy ARGS="--network sepolia"`

### DISCUSSION THREAD

**Discussion by LareineHan** [Cyfrin/foundry-full-course-cu#1348] [Q&A]
I ran `make deploy ARGS="--network sepolia"` and got this issue. 
Can anybody help me recognize the problem area and fix this issue?

<img width="804" alt="Screenshot 2024-01-23 at 2 14 20 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/7f2ddc91-b57e-4d69-9afd-b5eb16883999">

**Comment by 0xusmanf** [comment]
You need to add `0x` in the start of your private key in your .env file.

For example at the moment your private key is stored as below
```shell
PRIVATE_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```
You need to store it like in the example below
```shell
PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

**Reply by LareineHan** [reply]
Thanks! I changed my private key by adding `0x`, and everything went well, but I got an error like this.
`Error: 
Etherscan could not detect the deployment.`

Any clues on this issue?

**Reply by 0xusmanf** [reply]
Can you please elaborate on which step you received this issue? and also copy-paste the complete log.

**Reply by LareineHan** [reply]
This happened under the same command. `make deploy ARGS="--network sepolia"`
`lareinehan@Lareines-Air foundry-smart-contract-lottery-f23 % make deploy ARGS="--network sepolia"
[⠆] Compiling...
No files changed, compilation skipped
Traces:
  [8465442] DeployRaffle::run()
    ├─ [2551298] → new HelperConfig@0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141
    │   ├─ [0] VM::envUint("PRIVATE_KEY") [staticcall]
    │   │   └─ ← <env var value>
    │   └─ ← 23648 bytes of code
    ├─ [1121] HelperConfig::activeNetworkConfig() [staticcall]
    │   └─ ← 10000000000000000 [1e16], 30, VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 0x474e34a077df58807dbe9c96d3c009b23b3c6d0cce433e59bbf5b34f823bc56c, 5379, 500000 [5e5], 0x779877A7B0D9E8603169DdbD7836e478b4624789, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76]
    ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   └─ ← ()
    ├─ [605424] → new Raffle@0x53d4F4DAA70ce7694D1CA1125b5762556052190A
    │   └─ ← 3467 bytes of code
    ├─ [0] VM::stopBroadcast()
    │   └─ ← ()
    ├─ [5118188] → new AddConsumer@0xdaE97900D4B184c5D2012dcdB658c008966466DD
    │   └─ ← 25499 bytes of code
    ├─ [82095] AddConsumer::addConsumer(Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A], VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 5379, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   ├─ [0] console::log("Adding Consumer to Raffle ", Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("Using vrfCoordinator ", VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("On ChainID ", 11155111 [1.115e7]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   │   └─ ← ()
    │   ├─ [73410] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A])
    │   │   ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A])
    │   │   └─ ← ()
    │   ├─ [0] VM::stopBroadcast()
    │   │   └─ ← ()
    │   └─ ← ()
    └─ ← Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A], HelperConfig: [0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141]


Script ran successfully.

== Return ==
0: contract Raffle 0x53d4F4DAA70ce7694D1CA1125b5762556052190A
1: contract HelperConfig 0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141

== Logs ==
  Adding Consumer to Raffle  0x53d4F4DAA70ce7694D1CA1125b5762556052190A
  Using vrfCoordinator  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On ChainID  11155111

## Setting up 1 EVM.
==========================
Simulated On-chain Traces:

  [639098] → new DevOpsTools@0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d
    └─ ← 3250 bytes of code

  [605424] → new Raffle@0x53d4F4DAA70ce7694D1CA1125b5762556052190A
    └─ ← 3467 bytes of code

  [73410] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A])
    ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x53d4F4DAA70ce7694D1CA1125b5762556052190A])
    └─ ← ()


==========================

Chain 11155111

Estimated gas price: 94.030604265 gwei

Estimated total gas used for script: 2024023

Estimated amount required: 0.190320105736258095 ETH

==========================

###
Finding wallets for all the necessary addresses...
##
Sending transactions [0 - 2].
⠙ [00:00:00] [###############################################################################################################] 3/3 txes (0.0s)
Transactions saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/broadcast/DeployRaffle.s.sol/11155111/run-latest.json

Sensitive values saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/cache/DeployRaffle.s.sol/11155111/run-latest.json

##
Waiting for receipts.
⠚ [00:00:12] [###########################################################################################################] 3/3 receipts (0.0s)
##### sepolia
✅  [Success]Hash: 0x032515fe2c071c85f6d2a39586b9f801e14a0678359561c9942d1ef396d07964
Contract Address: 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d
Block: 5141708
Paid: 0.044154246239900896 ETH (742886 gas * 59.436099536 gwei)


##### sepolia
✅  [Success]Hash: 0x80304388cdf97348be31651c079b400f554e8feb74698b74ab3370390f378d47
Contract Address: 0x53d4F4DAA70ce7694D1CA1125b5762556052190A
Block: 5141708
Paid: 0.04205757839266896 ETH (707610 gas * 59.436099536 gwei)


##### sepolia
✅  [Success]Hash: 0x7372dc1ed40e5618edfa0f2843b8bc2918db4d5f61e54665604c5dbd00634290
Block: 5141708
Paid: 0.005646072839322784 ETH (94994 gas * 59.436099536 gwei)


Transactions saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/broadcast/DeployRaffle.s.sol/11155111/run-latest.json

Sensitive values saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/cache/DeployRaffle.s.sol/11155111/run-latest.json



==========================

ONCHAIN EXECUTION COMPLETE & SUCCESSFUL.
Total Paid: 0.09185789747189264 ETH (1545490 gas * avg 59.436099536 gwei)
##
Start verification for (2) contracts
Start verifying contract `0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d` deployed on sepolia

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Submitting verification for [lib/foundry-devops/src/DevOpsTools.sol:DevOpsTools] 0xB7fDCcD94e85F99f0d14Cb9442E8Bf376D6d231d.

Transactions saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/broadcast/DeployRaffle.s.sol/11155111/run-latest.json

Sensitive values saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/cache/DeployRaffle.s.sol/11155111/run-latest.json

Error: 
Etherscan could not detect the deployment.
make: *** [deploy] Error 1`

**Reply by 0xusmanf** [reply]
Sometimes it happens if the network is slow. You can use the following command and check if it verifies your contract.

```shell
forge script script/DeployRaffle.s.sol:DeployRaffle --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --verify --etherscan-api-key $ETHERSCAN_API_KEY -vvvv
```

**Reply by Blockchain-Oracle** [reply]
@LareineHan  your question is poorly formated , you have low chances of getting an answer

**Reply by LareineHan** [reply]
@usmanfarooq91  I got 
```
Error: 
Deployment not found for chain `11155111`.
```
<img width="902" alt="Screenshot 2024-01-27 at 8 26 46 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/6de27025-4876-4a6a-9bf8-2ef876304e8e">

after `forge script script/DeployRaffle.s.sol:DeployRaffle --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --verify --etherscan-api-key $ETHERSCAN_API_KEY -vvvv`

and I ran `make deploy ARGS="--network sepolia"` again, and I got this log. 
`lareinehan@Lareines-Air foundry-smart-contract-lottery-f23 % make deploy ARGS="--network sepolia"
[⠆] Compiling...
No files changed, compilation skipped
Traces:
  [8448342] DeployRaffle::run()
    ├─ [2551298] → new HelperConfig@0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141
    │   ├─ [0] VM::envUint("PRIVATE_KEY") [staticcall]
    │   │   └─ ← <env var value>
    │   └─ ← 23648 bytes of code
    ├─ [1121] HelperConfig::activeNetworkConfig() [staticcall]
    │   └─ ← 10000000000000000 [1e16], 30, VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 0x474e34a077df58807dbe9c96d3c009b23b3c6d0cce433e59bbf5b34f823bc56c, 5379, 500000 [5e5], 0x779877A7B0D9E8603169DdbD7836e478b4624789, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76]
    ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   └─ ← ()
    ├─ [605424] → new Raffle@0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531
    │   └─ ← 3467 bytes of code
    ├─ [0] VM::stopBroadcast()
    │   └─ ← ()
    ├─ [5118188] → new AddConsumer@0xdaE97900D4B184c5D2012dcdB658c008966466DD
    │   └─ ← 25499 bytes of code
    ├─ [64995] AddConsumer::addConsumer(Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531], VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 5379, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   ├─ [0] console::log("Adding Consumer to Raffle ", Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("Using vrfCoordinator ", VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("On ChainID ", 11155111 [1.115e7]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   │   └─ ← ()
    │   ├─ [56310] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531])
    │   │   ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531])
    │   │   └─ ← ()
    │   ├─ [0] VM::stopBroadcast()
    │   │   └─ ← ()
    │   └─ ← ()
    └─ ← Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531], HelperConfig: [0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141]


Script ran successfully.

== Return ==
0: contract Raffle 0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531
1: contract HelperConfig 0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141

== Logs ==
  Adding Consumer to Raffle  0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531
  Using vrfCoordinator  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On ChainID  11155111

## Setting up 1 EVM.
==========================
Simulated On-chain Traces:

  [639098] → new DevOpsTools@0x3A931E1bB8c034ABba70E5b3aA5CA50616734BC8
    └─ ← 3250 bytes of code

  [605424] → new Raffle@0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531
    └─ ← 3467 bytes of code

  [56310] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531])
    ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531])
    └─ ← ()


==========================

Chain 11155111

Estimated gas price: 98.107562294 gwei

Estimated total gas used for script: 1992685

Estimated amount required: 0.19549746776981939 ETH

==========================

###
Finding wallets for all the necessary addresses...
##
Sending transactions [0 - 2].

Transactions saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/broadcast/DeployRaffle.s.sol/11155111/run-latest.json

Sensitive values saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/cache/DeployRaffle.s.sol/11155111/run-latest.json

Error: 
(code: -32000, message: insufficient funds for gas * price + value: balance 109187538641122491, queued cost 94721477888984884, tx cost 90220597253623046, overshot 75754536501485439, data: None)
make: *** [deploy] Error 1`

Please advice to fix this. Thanks!

**Comment by LareineHan** [comment]
> @LareineHan your question is poorly formated , you have low chances of getting an answer
> I am doing my best as a learner, but thanks for your concerns @Blockchain-Oracle .

**Reply by Blockchain-Oracle** [reply]
This is mostly network issue, try running this command
```bash
forge clean
```

then try deploying again 

mark this as answer if that helps

**Reply by LareineHan** [reply]
@Blockchain-Oracle Thanks for your answer. I ran the command and then deployed it.
Here is the log after deploying. Ran into a similar error. Can you tell me what's the problem?

`lareinehan@Lareines-Air foundry-smart-contract-lottery-f23 % forge clean
lareinehan@Lareines-Air foundry-smart-contract-lottery-f23 %  make deploy ARGS="--network sepolia"
[⠒] Compiling...
[⠢] Compiling 33 files with 0.8.19
[⠰] Solc 0.8.19 finished in 4.15s
Compiler run successful with warnings:
Warning (5667): Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:306:45:
    |
306 |   function requestSubscriptionOwnerTransfer(uint64 _subId, address _newOwner) external pure override {
    |                                             ^^^^^^^^^^^^^

Warning (5667): Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:306:60:
    |
306 |   function requestSubscriptionOwnerTransfer(uint64 _subId, address _newOwner) external pure override {
    |                                                            ^^^^^^^^^^^^^^^^^

Warning (5667): Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:310:44:
    |
310 |   function acceptSubscriptionOwnerTransfer(uint64 _subId) external pure override {
    |                                            ^^^^^^^^^^^^^

Warning (5667): Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:314:33:
    |
314 |   function pendingRequestExists(uint64 subId) public view override returns (bool) {
    |                                 ^^^^^^^^^^^^

Warning (2018): Function state mutability can be restricted to pure
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:261:3:
    |
261 |   function getConfig()
    |   ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to pure
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:274:3:
    |
274 |   function getFeeConfig()
    |   ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to pure
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:302:3:
    |
302 |   function getFallbackWeiPerUnitLink() external view returns (int256) {
    |   ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to pure
   --> lib/chainlink-brownie-contracts/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol:314:3:
    |
314 |   function pendingRequestExists(uint64 subId) public view override returns (bool) {
    |   ^ (Relevant source part starts here and spans across multiple lines).

Traces:
  [8448342] DeployRaffle::run()
    ├─ [2551298] → new HelperConfig@0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141
    │   ├─ [0] VM::envUint("PRIVATE_KEY") [staticcall]
    │   │   └─ ← <env var value>
    │   └─ ← 23648 bytes of code
    ├─ [1121] HelperConfig::activeNetworkConfig() [staticcall]
    │   └─ ← 10000000000000000 [1e16], 30, VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 0x474e34a077df58807dbe9c96d3c009b23b3c6d0cce433e59bbf5b34f823bc56c, 5379, 500000 [5e5], 0x779877A7B0D9E8603169DdbD7836e478b4624789, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76]
    ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   └─ ← ()
    ├─ [605424] → new Raffle@0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1
    │   └─ ← 3467 bytes of code
    ├─ [0] VM::stopBroadcast()
    │   └─ ← ()
    ├─ [5118188] → new AddConsumer@0xdaE97900D4B184c5D2012dcdB658c008966466DD
    │   └─ ← 25499 bytes of code
    ├─ [64995] AddConsumer::addConsumer(Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1], VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625], 5379, 18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   ├─ [0] console::log("Adding Consumer to Raffle ", Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("Using vrfCoordinator ", VRFCoordinatorV2: [0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] console::log("On ChainID ", 11155111 [1.115e7]) [staticcall]
    │   │   └─ ← ()
    │   ├─ [0] VM::startBroadcast(18002654368769567123658957798224351208121609070851417164401623809763700531496 [1.8e76])
    │   │   └─ ← ()
    │   ├─ [56310] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1])
    │   │   ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1])
    │   │   └─ ← ()
    │   ├─ [0] VM::stopBroadcast()
    │   │   └─ ← ()
    │   └─ ← ()
    └─ ← Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1], HelperConfig: [0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141]


Script ran successfully.

== Return ==
0: contract Raffle 0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1
1: contract HelperConfig 0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141

== Logs ==
  Adding Consumer to Raffle  0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1
  Using vrfCoordinator  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On ChainID  11155111

## Setting up 1 EVM.
==========================
Simulated On-chain Traces:

  [639098] → new DevOpsTools@0x260Ead3b8fA7284Ad763745Ef0beB0C7AF97D531
    └─ ← 3250 bytes of code

  [605424] → new Raffle@0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1
    └─ ← 3467 bytes of code

  [56310] VRFCoordinatorV2::addConsumer(5379, Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1])
    ├─ emit SubscriptionConsumerAdded(subId: 5379, consumer: Raffle: [0x2A9A3Af45957d3b35169Ff8724837aD684d85AA1])
    └─ ← ()


==========================

Chain 11155111

Estimated gas price: 53.800469444 gwei

Estimated total gas used for script: 1992685

Estimated amount required: 0.10720738845401714 ETH

==========================

###
Finding wallets for all the necessary addresses...
##
Sending transactions [0 - 2].

Transactions saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/broadcast/DeployRaffle.s.sol/11155111/run-latest.json

Sensitive values saved to: /Users/lareinehan/foundry-f23/foundry-smart-contract-lottery-f23/cache/DeployRaffle.s.sol/11155111/run-latest.json

Error: 
(code: -32000, message: replacement transaction underpriced, data: None)
make: *** [deploy] Error 1`

**Comment by Blockchain-Oracle** [comment]
confirm your private key accounts if you have enough funds .. 
if no, you can head up to [this link for sepolia eth](https://faucets.chain.link/)
